docs/CONNECTOR.md

```diff
@@ -0,0 +1,51 @@
+# Connector toolkit
+This library includes a base toolkit to create a ship - connector to integrate Hull platform with a selected 3rd party API Service.
+
+## new Hull.App()
+```js
+import Hull from "hull";
+
+const app = new Hull.App({ port, hostSecret, service });
+
+// ... application configuration
+
+app.start();
+
+```
+
+### server()
+```js
+const app = new Hull.App({ port, hostSecret, service });
+
+const server = app.server();
```

Comment:
> we might want to add middlewares before and after hull does it's job

This is still the hardest point of the architecture. Before we had the handlers/routers a blackboxes with all middlewares being applied (https://github.com/hull/hull-node/blob/master/src/notif-handler.js#L173), without the option to inject some custom logic between selected middlewares. Right now, they are split into a app level and router level, this allows for some degree of customization, but still not perfect.

I will come up with some new examples here too.